README.md

```diff
@@ -255,14 +233,24 @@ Perform following commands
 
 # NodeJS
 
-## Practical 15
+## Practical 13
 
 - Real-time group chat application
-  
-  - Build a real-time group chat application using Node.js, Javascript, and a library like Socket.IO.
+  - Build a real-time group chat application using Node.js, TypeScript, and a library like Socket.IO.
   - Allow users to join chat rooms, send messages, and receive messages in real time.
 
-## Practical 16
+## Practical 14
+- Quiz Application with Firebase Realtime Database
```

Comment:
It will add extra complexity in  designing schema as per candidate level. I think it would be better to add chat application like whatsapp.

Reply:
We have already one chat application in the practical and one in the evaluation test. I think a simplified quiz application will be fine.